server.js

```diff
@@ -8,19 +8,32 @@ const db = require('./server/db/db');
 // Get our API routes
 const api = require('./server/routes/api');
 const contact = require('./server/routes/contactus.route');
+const user = require('./server/routes/user.route');
 
 const app = express();
 
+
 // Parsers for POST data
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
+//to allow Cross origin requests
+app.use(function(req, res, next) {
+  res.setHeader("Access-Control-Allow-Origin", "*");
```

Comment:
`"Access-Control-Allow-Origin", "*"` don't allow this. This header shouldn't be there in the first place, use some kind of reverse proxy to mitigate `CORS`

Reply:
https://www.npmjs.com/package/cors-anywhere can I use this as an alternate?

Reply:
You might not need anything such as this package

Reply:
`CORS` is not required. For development environment use proxying present in `angular-cli`. We can discuss the deployment with @afirdousi

Reply:
I understand now. Thanks

Reply:
I agree to @mohuk. I don't think we would need CORS.